debian/control

```diff
@@ -0,0 +1,25 @@
+Source: novnc
+Section: web
+Priority: optional
+Maintainer: pi-top <deb-maintainers@pi-top.com>
+Build-Depends:
+ debhelper-compat (= 12),
+Standards-Version: 4.5.1
+Homepage: https://novnc.com
+
+Package: novnc
+Architecture: all
+Depends:
+ ${misc:Depends},
+ procps,
```

Comment:
We previously also found that novnc uses `netstat` command which is from `net-tools` package. Like procps this is included in common distributions of debian but not all (eg wsl, docker)

Reply:
There are other things [here](https://salsa.debian.org/openstack-team/third-party/novnc/-/blob/debian/yoga/debian/control), maybe take another look at that

Reply:
`novnc_proxy` is more simple than `launch.sh` so no other dependencies are required (other than `procps`)... just added `python3-novnc` and `python3-numpy` just to be safe